packages/expo-video/android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <application>
+  <activity
+    android:name=".FullscreenActivity"
+    android:configChanges="orientation|keyboardHidden|screenSize"
```

Comment:
Do we need the `configChanges` property?

Reply:
Fullscreen seems to work fine without it, although most guides suggested using this property for fullscreen. I guess we can add it back if we encounter any issues.